neuro_san/http_sidecar/handlers/concierge_handler.py

```diff
@@ -43,4 +43,4 @@ def get(self):
             self.process_exception(exc)
         finally:
             self.do_finish()
-            self.logger.info(metadata, "Finish GET /api/v1/list")
+            self.application.finish_client_request(metadata, "/api/v1/list")
```

Comment:
Here and in all request handlers:
frame request logic with start_client_request() and finish_client_request().
These "application" (http server) level calls will provide request logging and stats counting.